README.md

```diff
@@ -89,58 +92,275 @@ Now you can import like this:
 const zbench = @import("zbench");
 ```
 
+### Compatibility Notes
+
+***Supported Version***: As of now, this fork of zBench is tested and supported on Zig version ***0.12.0-dev.1849+bb0f7d55e***.
+
```

Comment:
a general remark on Zig versions; the longer I use Zig, the more I tend towards using the latest stable version (not the master/dev version) for library code. Zig is evolving relatively fast, with breaking changes in the master coming without warning (!). Changing from one stable version to another will likely require some serious rewriting, but I believe this is better than *maybe* having to rewrite a lot of code every other day after updating Zig.